File: OOP_Lab/lab1.cpp
#include <bits/stdc++.h>
using namespace std;
class Traigle
{
public:
    double width;
    double height;
    Traigle(double width, double height)
    {
        this->height = height;
        this->width = width;
    }
    double area()
    {
        return .5 * width * height;
    }
};
int main()
{
    int h, w;
    cout << "Enter the Height and Widgh :" << endl;
    cin >> h >> w;
    Traigle *area = new Traigle(h, w);
    double are1 = area->area();
    cout << "Traigle area is : " << are1 << endl;

    return 0;
}

File: OOP_Lab/lab1.java
import java.util.Scanner;

class Traigle {
    public double height;
    public double wight;

    Traigle(double height, double wight) {
        this.height = height;
        this.wight = wight;
    }

    double area() {
        return .5 * height * wight;
    }

}

public class lab1 {
    public static void main(String[] args) {
        Scanner intput = new Scanner(System.in);

        System.out.println("Enter the Height and width: ");
        double h = intput.nextDouble();
        double w = intput.nextDouble();
        Traigle T = new Traigle(h, w);
        double area = T.area();
        System.out.println("This araea of the Traigle is: " + area);
    }

}